vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java

```diff
@@ -215,13 +215,16 @@ public void testSafetySettings() throws Exception {
 
   @Test
   public void testTokenCount() throws Exception {
-    String textPrompt = "Why is the sky blue?";
-
-    int tokenCount =
-        GetTokenCount.getTokenCount(PROJECT_ID, LOCATION, GEMINI_PRO_VISION, textPrompt);
+    int tokenCount = GetTokenCount.getTokenCount(PROJECT_ID, LOCATION, GEMINI_FLASH);
     assertThat(tokenCount).isEqualTo(6);
   }
 
+  @Test
+  public void testMediaTokenCount() throws Exception {
+    int tokenCount = GetMediaTokenCount.getMediaTokenCount(PROJECT_ID, LOCATION, GEMINI_FLASH);
+    assertThat(tokenCount).isEqualTo(16822);
```

Comment:
Is this number the same every time? Wondering if this might cause flaky issues...

Reply:
It's stable, but maybe it'll change when we change the model used.
It's hard to predict, but for a fixed version of a model, it's stable.